cime_config/acme/machines/template.st_archive

```diff
@@ -16,53 +127,58 @@ from CIME.case_submit               import submit
 from CIME.XML.standard_module_setup import *
 from CIME.case                      import Case
 from CIME.utils                     import get_model, run_cmd, appendStatus
+from CIME.st_archive                import short_term_archive
 
 logger = logging.getLogger(__name__)
 
-# Batch system directives
-{{ batchdirectives }}
-
 ###############################################################################
-def parse_command_line(args):
+def parse_command_line(args, description):
 ###############################################################################
 
     get_model()
 
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(
+        description=description,
+        formatter_class=argparse.RawDescriptionHelpFormatter
+    )
 
     CIME.utils.setup_standard_logging_options(parser)
 
     parser.add_argument("--caseroot", default=os.getcwd(),
                         help="Case directory to build")
 
+    parser.add_argument("--input", "-in", action="store_true",
```

Comment:
There may be a problem here - it's not easy to pass command line arguments to a script submitted to the batch system - are these arguments intended only for using the script from the command line?